src/routers/opportunitiesRouter.ts

```diff
@@ -1,11 +1,11 @@
 import express from 'express';
 import { opportunitiesHandlers } from '../handlers/opportunitiesHandlers';
-import { checkApiKey } from '../middleware/apiKeyChecker';
+import { verifyJwt } from '../middleware/verifyJwt';
 
 const opportunitiesRouter = express.Router();
 
-opportunitiesRouter.post('/', checkApiKey, opportunitiesHandlers.postOpportunity);
-opportunitiesRouter.get('/:id', checkApiKey, opportunitiesHandlers.getOpportunity);
-opportunitiesRouter.get('/', checkApiKey, opportunitiesHandlers.getOpportunities);
+opportunitiesRouter.post('/', verifyJwt, opportunitiesHandlers.postOpportunity);
+opportunitiesRouter.get('/:id', verifyJwt, opportunitiesHandlers.getOpportunity);
+opportunitiesRouter.get('/', verifyJwt, opportunitiesHandlers.getOpportunities);
```

Comment:
Some of the routers have a `thingRouter.use(verifyJwt)` and some have `thingRouter.method(path, verifyJwt, ...)`. Why the split? Can we just use `use` everywhere?